ios/Classes/PosthogFlutterPlugin.swift

```diff
@@ -44,6 +47,9 @@ public class PosthogFlutterPlugin: NSObject, FlutterPlugin {
          postHogSdkName = "posthog-flutter"
          postHogVersion = postHogFlutterVersion
         
+        NotificationCenter.default.addObserver(forName: PostHogSDK.didReceiveFeatureFlags, object: nil, queue: nil)  { (notification) in
```

Comment:
Similar to https://github.com/PostHog/posthog-flutter/pull/111/files#r1624601103
but in this case `didReceiveFeatureFlags` is called even if there's an error, we'd probably need something more so we know if feature flags are errored or not.